stages/org.osbuild.ovf

```diff
@@ -22,7 +22,7 @@ OVF_TEMPLATE = """<?xml version="1.0"?>
   <VirtualSystem ovf:id="image">
     <Info>A virtual machine</Info>
     <Name>VM</Name>
-    <OperatingSystemSection ovf:id="100" vmw:osType="other26xLinux64Guest">
+    <OperatingSystemSection ovf:id="100" vmw:osType={"ostype"}>
```

Comment:
```suggestion
    <OperatingSystemSection ovf:id="100" vmw:osType={ostype}>
```
need to drop the quotes.

Reply:
Nice catch, it seems the string is also not an `f-string` so that would need changing too.

Reply:
Oh, nevermind, sorry, there is a OVF_TEMPLATE.format() in the code. I think we still want a test for this, something like:

[0001-stages-ovf-add-unit-test-for-ostype-template-writing.patch.txt](https://github.com/user-attachments/files/16658207/0001-stages-ovf-add-unit-test-for-ostype-template-writing.patch.txt)

Reply:
I have added test you proposed extended existing one. Still I do not know why it is failing :/

Reply:
There are quotes missing around `osType={ostype}` which means it's not valid xml. This `vmw:osType="{ostype}">` probably works